react-ui-kit/src/Inputs/Text/utils-style.ts

```diff
@@ -1,10 +1,6 @@
-import { ITheme } from '../../palette/variables';
-import { Status } from '../common';
+import type { ITheme } from '../../palette';
+import type { Status } from '../common';
 
 export const getIconColor = (errorActive = false, status: Status, theme: ITheme) => {
-  if (status === 'error' && errorActive) return theme.colors.neutral[50];
-
-  if (status === 'error') return theme.colors.error[60];
-
-  return theme.colors.neutral[50];
+  return status === 'error' && errorActive ? theme.colors.neutral[50] : theme.colors.error[60];
```

Comment:
Может всё же:
```suggestion
  return status === 'error' && !errorActive ? theme.colors.error[60] : theme.colors.neutral[50];
```
как я писал в предыдущем комментарии?

Из старого кода следует, что мы получаем цвет **theme.colors.error[60]** только в случае если `status === 'error'` и `errorActive === false`, во всех истальных случаях *theme.colors.neutral[50]*.

Ты переписал на следующее условие: мы получаем *theme.colors.neutral[50]* толькол в случае если `status === 'error'` и `errorActive === true`, во всех остальных **theme.colors.error[60]**.

Может там что-то ещё поменялось и теперь требуется именно такое условие, но обрати на это внимание, плиз